src/ui_utils.c

```diff
@@ -2477,10 +2477,24 @@ void ui_init_builder(void)
 static void init_custom_style(void)
 {
 #if GTK_CHECK_VERSION(3, 0, 0)
-	gchar *css_file = g_build_filename(app->datadir, "geany.css", NULL);
+	const struct {
+		guint version;
+		const gchar *file;
+	} css_files[] = {
+		{ 20, "geany-3.20.css" },
+		{ 0, "geany-3.0.css" },
+	};
```

Comment:
The braces on their own line is certainly the style used in code, but when
I looked at other declarations with initializations they seemed to use this
style already, maybe I was just (un?)lucky with my samples :)